networks.json

```diff
@@ -3,27 +3,405 @@
     "4": {
       "events": {},
       "links": {},
-      "address": "0xE091F109b721A4A42973595FfEE621c04A2147D1",
-      "transactionHash": "0x172014a549bf2639d98caf903671e2e8bccf5a4734241b797f18b70d690225e9"
+      "address": "0xfb0cbD725B3970239E0437D92A04b7676992C3be",
+      "transactionHash": "0x19f272db8dd11c989d37f2459dc1c8b84ef75959e8c382b2bd1943ce8af64493"
     }
   },
   "SnappAuction": {
     "4": {
       "links": {
         "IdToAddressBiMap": "0x31172bB2B5f97E8E89cf3376495d7Bc7252f5a53",
-        "SnappBaseCore": "0x3709B8E6E2a684A1c97809E616877A6577e9e131"
+        "SnappBaseCore": "0x391F9a3A6A8752B9feF5f6b13B701c55A2514237"
```

Comment:
Did this address really change? Is there some way to deploy only StableX contracts?